docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -325,16 +329,134 @@ When the proxy datasource is obtained via a JNDI lookup by the JDBC persistence
 
 The proxy datasource relies on the presence of the tenant-id and datastore-id information within the thread-local `FHIRRequestContext` information. In a simple configuration of the FHIR server that involves the use of the JDBC persistence layer implementation, the `FHIRRequestContext` information is obtained via request headers from the incoming REST API request. The tenant-id is obtained from the `X-FHIR-TENANT-ID` request header and the datastore-id is obtained from the `X-FHIR-DSID` request header<sup id="a3">[3](#f3)</sup>.
 
-#### 3.4.2.2 Datastore configuration examples
+#### 3.4.2.2 Standard JDBC Datasources
+
+To use standard JDBC datasources, disable the proxy datasource behavior by setting the 'enableProxyDatasource' flag to false (default is true):
+
+```
+    {
+        "fhirServer": {
+            …
+            "persistence": {
+                …
+                "jdbc": {
+                    …
+                    "enableProxyDatasource": false
+                }
+                …
+            }
+    }
+```
+
+The 'dataSourceJndiName' property can be removed as it is only used when 'enableProxyDatasource' is true. The 'connectionProperties' element can also be removed (these properties are defined in the Liberty JDBC datasource definition). By default, the JNDI name of the datasource is 'jdbc/fhir_<tenantId>_<dsId>' where '<tenantId>' and '<dsId>' represent the tenant and datastore ids respectively. The JNDI address of the datasource can also be provided explicitly using the 'jndiName' property as shown in the following example:
+
+    ```
+    {
+        "fhirServer":{
+            …
+            "persistence":{
+                "datasources": {
+                    "default": {
+                        "tenantKey": "<the-base64-tenant-key>",
+                        "jndiName": "jdbc/fhir_default_default",
+                        "type": "db2",
+                        "hints" : {
+                            "search.reopt": "ONCE"
+                        }
+                    }
+                }
+            …
+            }
+        }
+    }
+    ```
```

Comment:
please shift the Markdown code block left. avoiding gatsby compilation issues.

Reply:
done